client/login/qr-code-login/index.jsx

```diff
@@ -0,0 +1,50 @@
+import { Card, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import QRCode from 'qrcode.react';
+import qrCenter from 'calypso/assets/images/qr-login/wp.png';
```

Comment:
I actually wonder if we could use the `wordpress-logo` component with a few additional styles for this, instead of having to introduce a new image 🤔

Reply:
The QRCode react component requires an image URL so a WordPress-logo component doesn't work.